commands/files/serialfile.go

```diff
@@ -42,21 +36,11 @@ func newSerialFile(name, path string, file *os.File, stat os.FileInfo) (File, er
 
 	// for directories, stat all of the contents first, so we know what files to
 	// open when NextFile() is called
-	contents, err := file.Readdir(0)
+	contents, err := ioutil.ReadDir(path)
```

Comment:
Not possible because in `NextFile()`, `f.Close()` requires the opened `os.File` object.